### PR TITLE
func_802A5290

### PR DESCRIPTION
**Pull request by ethteck**
and some work on other funcs + improvement to find_similar_areas.py

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀 This PR matches 1 function (+0.03%) on `us`.

File: src/80850_len_3060.c [hunk 1]
```diff
@@ -506,30 +506,43 @@ void status_menu_draw_stat(s32 id, s32 x, s32 y, s32 currentValue, s32 maxValue)
...
+        regalloc:
+        target  |  current     |  do while
+        ------------------------------------------
+        s5      |  sp54        |   s7
+        s6      |  s5          |   s5
+        s7      |  s6          |   s6
+        s8      |  sp50        |   sp50
+        sp50    |  s8          |   sp54
+        sp54    |  s7          |   s8
+    */
```
Comment (nanaian) [inline src/80850_len_3060.c:545]: Probably can remove this now its matched?

**Review by nanaian — APPROVED** [review]
awesome do..while(0) work

File: src/80850_len_3060.c (hunk 1 above)
Comment (ethteck) [reply src/80850_len_3060.c:545]: that one isn't matched - it's another one. the do while 0 stuff is wip work on that function that unnunu found I think :)

**Pull request merged**